src/components/page_components/main/MovieContainer.jsx

```diff
@@ -3,12 +3,19 @@ import MovieItem from './MovieItem';
 import { useEffect, useState } from 'react';
 export default function MovieContainer() {
   const sortList = [
-    { eng: 'favoite', kor: '찜한 영화' },
-    { eng: 'release', kor: '개봉순' },
-    { eng: 'popularity', kor: '인기순' },
-    { eng: 'title', kor: '제목순' },
+    { key: 'favoite', label: '찜한 영화' },
+    { key: 'release', label: '개봉순' },
+    { key: 'popularity', label: '인기순' },
+    { key: 'title', label: '제목순' },
   ];
+  const [checked, setChecked] = useState([0, 0, 0, 0]);
   const [data, setData] = useState([]);
+
+  function sortChecked(id) {
+    checked[id] = checked[id] === 0 ? 30 : 0;
+    setChecked([...checked]);
+  }
```

Comment:
해당 코드를 보고, 저는 객체로 하는걸 생각해 봤어요😊

이렇게 코드를 짠다면, `삼항연산자`를 사용하는 것 보다 직관적으로 볼 수 있을 것 같습니다!
```suggestion
  const [checked, setChecked] = useState({
    0 : 0,
    1: 0,
    2: 0,
    3: 0,
  });
  const [data, setData] = useState([]);

  function sortChecked(id) {
    setChecked((prev)=>{
      return {
        ...prev,
        [id] : 30,
      }
    })
    //checked[id] = checked[id] === 0 ? 30 : 0;
    //setChecked([...checked]);
  }
```

Reply:
리뷰 감사합니다! 근데 제가 원한 기능은 한번 눌렀을때는 살짝 위로 올라가고 다시 눌렀을때는 내려가기를 원했는데
삼항연산자를 쓰지 않으면 다시 눌렀을때 해당 id의 값이 0이 안되지 않나요?

Reply:
그렇네요.
```
[id] : checked[id] === 0 ? 30 : 0,
```
이렇게 넣을 수 있긴 할텐데 이러면 결국 로직이 똑같네요ㅎㅎ